packages/contracts/contracts/BorrowerOperations.sol

```diff
@@ -144,14 +149,14 @@ contract BorrowerOperations is LiquityBase, BorrowerOperationsStorage, CheckCont
         masset = IMasset(_massetAddress);
     }
 
-    function openTrove(uint _maxFeePercentage, uint _ZUSDAmount, address _upperHint, address _lowerHint) external payable override {
-        _openTrove(_maxFeePercentage, _ZUSDAmount, _upperHint, _lowerHint, msg.sender, msg.value, msg.sender);
+    function openTrove(uint _maxFeePercentage, uint _ZUSDAmount, address _upperHint, address _lowerHint, uint _amount) external override {
+        _openTrove(_maxFeePercentage, _ZUSDAmount, _upperHint, _lowerHint, msg.sender, _amount, msg.sender);
     }
 
-    function openNueTrove(uint _maxFeePercentage, uint _ZUSDAmount, address _upperHint, address _lowerHint) external payable override {
+    function openNueTrove(uint _maxFeePercentage, uint _ZUSDAmount, address _upperHint, address _lowerHint, uint _amount) external override {
```

Comment:
This is fine although I'm not sure it's required. I will double check with @john-light

Reply:
@swamp-thing-sovryn did we already discuss this? If not, what does this change do?

Reply:
Opens a trove and sends the stable coin directly to `Mynt`

Reply:
Correct. This won't be needed for the SOV version, because Mynt is only for BTC-backed stablecoins. However, we may want to replace the Mynt integration with a BabelFish XUSD integration. Let me get back to you about this.

Reply:
To close off this thread: due to resource and time constraints, a decision was made to disable the aggregator integration for now.